### PR TITLE
Rent endpoints

### PR DESCRIPTION
**Pull request by ffasitella**
### Summary

Endpoints to:
- Create a new rent
- Get all the users' rents

### Trello card

https://trello.com/c/POyMV8km/28-crear-endpoint-de-creacion-de-rent-e-index

### CODE REVIEW

File: app/controllers/api/v1/rents_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      include Wor::Paginate
+
+      before_action :authenticate_user!
+
+      def index
+        render_paginated @current_user.rents, each_serializer: RentSerializer
+      end
+
+      def create
+        @rent = Rent.create(rent_params)
```
Comment (edkajimenez) [inline app/controllers/api/v1/rents_controller.rb]: why do you declare this as an instance variable? do you need to use it out of the controller's scope? like on a view or mailer

Comment (ffasitella) [reply app/controllers/api/v1/rents_controller.rb]: I will use it later (in the next PRs) in the rent mailer, should I modify this?

Comment (edkajimenez) [reply app/controllers/api/v1/rents_controller.rb]: If you are going to use this in a mailer its probably ok, I would let it just like that

File: app/controllers/api/v1/rents_controller.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      include Wor::Paginate
```
Comment (marcegl91) [inline app/controllers/api/v1/rents_controller.rb]: This code seems repeated in BooksController and RentsController. Maybe we can have RentsController and BooksController to subclassify from an ApiController that has all the logic that is repeated.

File: app/controllers/api/v1/books_controller.rb [hunk 3]
```diff
@@ -1,8 +1,6 @@
 module Api
   module V1
-    class BooksController < ApplicationController
-      include Wor::Paginate
-
+    class BooksController < ApiController
       before_action :authenticate_user!
```
Comment (marcegl91) [inline app/controllers/api/v1/books_controller.rb]: I think every endpoint should be authenticated in this training, so you can move this line to `ApiController` and remove it from both controllers.

**Pull request merged**